apps/www/src/components/ui/organisms/content-sections/content-alternate-position-with-image-list/content-alternate-position-with-image-list.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import {faker} from '@faker-js/faker'
+import {imageFactory} from '@/tests/factories/image.factory'
+import ContentAlternatePositionWithImageList, {ContentAlternatePositionWithImageListProps} from './index'
+import type {Meta} from '@storybook/react'
+
+const meta = {
+  title: 'components/Organisms/ContentSections/ContentAlternatePositionWithImageList',
+  component: ContentAlternatePositionWithImageList,
+  tags: ['autodocs'],
+  args: {
+    items: Array.from({length: 3}, () => ({
```

Comment:
Create a factory for ContentAlternatePositionWithImageItem so you don't need to `hardcode` it here and you can use the factory for `content-alternate-position-with-image-item.stories.tsx`